openshift/ci-operator/static-images/dispatcher/hermetic/Dockerfile.deps

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG JAVA_BUILDER=registry.access.redhat.com/ubi8/openjdk-21
+
+FROM $JAVA_BUILDER
+
+USER root
+
+WORKDIR /build
+
+COPY /data-plane/pom.xml .
+COPY /data-plane/core/pom.xml core/pom.xml
+COPY /data-plane/receiver/pom.xml receiver/pom.xml
+COPY /data-plane/receiver-loom/pom.xml receiver-loom/pom.xml
+COPY /data-plane/dispatcher/pom.xml dispatcher/pom.xml
+COPY /data-plane/dispatcher-loom/pom.xml dispatcher-loom/pom.xml
+COPY /data-plane/contract/pom.xml contract/pom.xml
+
+RUN mvn package dependency:go-offline -Drelease -DskipTests -Dmaven.repo.local=/third_party/maven
```

Comment:
Don't we need a patch for the `data-plane/pom.xml` to add the dependency plugin? (Don't see it in upstreams: https://github.com/knative-extensions/eventing-kafka-broker/blob/55b62644533231cda332bf6f8d5aa2feaca78bc3/data-plane/pom.xml#L430)